src/less/donut-chart/stories/donut-chart.stories.js

```diff
@@ -0,0 +1,55 @@
+        <path class="donut-chart__segment donut-chart__segment--quinary" d="M 9.113284184547226 139.55650989666063 A 111 111 0 0 1 9.151597969594818 96.25085345519774 Q 10.343988814188137 90.9595995673076 13.05963594319961 91.64169975840255 L 21.206577330234026 93.68800033168738 Q 23.9222244592455 94.37010052278232 22.880225252708982 98.99398905544308 A 97 97 0 0 0 22.846743836946672 136.8376708105953 Q 23.880559638903847 141.46339587770842 21.163709814047465 142.14068977933297 L 13.013160339478318 144.1725714842067 Q 10.296310514621936 144.84986538583127 9.113284184547226 139.55650989666063" data-id="4" role="graphics-symbol" tabindex="-1" aria-labelledby="chart-legend-label-4"></path>
+        <path class="donut-chart__segment donut-chart__segment--septenary" d="M 17.646455457719867 70.56408430523769 A 111 111 0 0 1 34.432567172848394 44.94191235270729 Q 38.101205973650934 40.946883819438895 40.116671048189474 42.890566029399 L 46.16306627180508 48.7216126592793 Q 48.17853134634362 50.66529486923939 44.97260374564229 54.15644592984331 A 97 97 0 0 0 30.303659273863303 76.54699259106357 Q 28.38338679108672 80.8804278450838 25.796515481963453 79.808935040612 L 18.035901554593636 76.59445662719656 Q 15.449030245470368 75.52296382272476 17.646455457719867 70.56408430523769" data-id="5" role="graphics-symbol" tabindex="-1" aria-labelledby="chart-legend-label-5"></path>
+        <path class="donut-chart__segment donut-chart__segment--senary" d="M 54.589228124478055 26.895258025992334 A 111 111 0 0 1 64.30329669944926 20.852359500332568 Q 69.11304306300369 18.345268845677637 70.3462275623153 20.859081883804677 L 74.04578106025016 28.40052099818584 Q 75.27896555956178 30.914334036312894 71.0758538724917 33.10521505884918 A 97 97 0 0 0 62.58698313580515 38.3859462028942 Q 58.764010466953565 41.187907566316014 57.054105614453256 38.97065128987978 L 51.92439105695233 32.31888246057106 Q 50.214486204452015 30.101626184134815 54.589228124478055 26.895258025992334" data-id="6" role="graphics-symbol" tabindex="-1" aria-labelledby="chart-legend-label-6"></path>
+        <path class="donut-chart__segment donut-chart__segment--octanary" d="M 89.40045961508001 10.747651355453613 A 111 111 0 0 1 91.14666958843688 10.297174383364393 Q 96.43998701458307 9.113977756239734 96.98384319799898 11.860652191217468 L 98.61541174824676 20.100675496150686 Q 99.15926793166267 22.84734993112842 94.53357612683223 23.881314551228343 A 97 97 0 0 0 93.00760885281767 24.27497460791892 Q 88.45900042086535 25.607741970089464 87.60627053610682 22.94074895479308 L 85.04808088183124 14.939769908903926 Q 84.19535099707272 12.272776893607542 89.40045961508001 10.747651355453613" data-id="7" role="graphics-symbol" tabindex="-1" aria-labelledby="chart-legend-label-7"></path>
+    </svg>
+</div>
+`;
+
+export const patterFill = () => `
```

Comment:
Patterns look to be missing. 
<img width="530" alt="image" src="https://user-images.githubusercontent.com/1675667/199601942-a3a15a45-07b3-47d3-8fb2-438adac3ceaa.png">